frontend/src/container/MetricsApplication/Tabs/External.tsx

```diff
@@ -106,6 +112,11 @@ function External(): JSX.Element {
 		[servicename, tagFilterItems],
 	);
 
+	externalCallErrorWidget.title = EXTERNAL_CALL_ERROR_PERCENTAGE;
```

Comment:
can we directly use the title here ?

Reply:
Okay, I figured it out, where we can use this.